path_role.go

```diff
@@ -16,83 +16,92 @@ import (
 // pathsRole returns the path configurations for the CRUD operations on roles
 func pathsRole(b *azureAuthBackend) []*framework.Path {
 	p := []*framework.Path{
-		&framework.Path{
+		{
 			Pattern: "role/?",
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ListOperation: b.pathRoleList,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ListOperation: &framework.PathOperation{
+					Callback: b.pathRoleList,
+				},
 			},
 			HelpSynopsis:    strings.TrimSpace(roleHelp["role-list"][0]),
 			HelpDescription: strings.TrimSpace(roleHelp["role-list"][1]),
 		},
-		&framework.Path{
+		{
 			Pattern: "role/" + framework.GenericNameRegex("name"),
 			Fields: map[string]*framework.FieldSchema{
-				"name": &framework.FieldSchema{
+				"name": {
 					Type:        framework.TypeString,
 					Description: "Name of the role.",
 				},
-				"policies": &framework.FieldSchema{
+				"policies": {
 					Type:        framework.TypeCommaStringSlice,
 					Description: tokenutil.DeprecationText("token_policies"),
 					Deprecated:  true,
 				},
-				"num_uses": &framework.FieldSchema{
+				"num_uses": {
 					Type:        framework.TypeInt,
 					Description: tokenutil.DeprecationText("token_num_uses"),
 					Deprecated:  true,
 				},
-				"ttl": &framework.FieldSchema{
+				"ttl": {
 					Type:        framework.TypeDurationSecond,
 					Description: tokenutil.DeprecationText("token_ttl"),
 					Deprecated:  true,
 				},
-				"max_ttl": &framework.FieldSchema{
+				"max_ttl": {
 					Type:        framework.TypeDurationSecond,
 					Description: tokenutil.DeprecationText("token_max_ttl"),
 					Deprecated:  true,
 				},
-				"period": &framework.FieldSchema{
+				"period": {
 					Type:        framework.TypeDurationSecond,
 					Description: tokenutil.DeprecationText("token_period"),
 					Deprecated:  true,
 				},
-				"bound_subscription_ids": &framework.FieldSchema{
-					Type: framework.TypeCommaStringSlice,
-					Description: `Comma-separated list of subscription ids that login 
-is restricted to.`,
+				"bound_subscription_ids": {
+					Type:        framework.TypeCommaStringSlice,
+					Description: `Comma-separated list of subscription ids that login is restricted to.`,
 				},
-				"bound_resource_groups": &framework.FieldSchema{
+				"bound_resource_groups": {
 					Type: framework.TypeCommaStringSlice,
 					Description: `Comma-separated list of resource groups that login 
-is restricted to.`,
+						is restricted to.`,
```

Comment:
@elsesiy Looks like there are still a few indents left here.

Nit: you don't have to force-push since the PR will get squash-merged anyways, and it makes the diff'ed changes in between commits a bit easier to look at.

Reply:
Okay yeah I didn't know what the merge strategy is for hashicorp repos, hence always squashed & rebased locally.